modules/apps/frontend-theme/frontend-theme-classic/src/templates/portal_normal.ftl

```diff
@@ -90,7 +90,7 @@
 					<div class="row">
 						<div class="col-md-12 text-center text-md-left">
 							<@liferay.language_format
-								arguments='<a class="text-white" href="http://www.liferay.com" rel="external">Liferay</a>'
+								arguments='<a class="text-white" href="http://www.liferay.com" rel="external" style="text-decoration: underline;">Liferay</a>'
```

Comment:
@ericthmoritsuka instead of using `style` here you can use the `text-decoration-underline` class which will handle that for you.